apps/facilities/test/repo_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Facilities.RepoTest do
+  use ExUnit.Case
+  alias Facilities.Repo
+
+  import Mock
+
+  describe "get_for_stop/1" do
+    test "should call the api" do
+      with_mock V3Api.Facilities, filter_by: fn _stop_id -> [] end do
+        assert [] = Repo.get_for_stop("test-id")
+        assert_called(V3Api.Facilities.filter_by([{"stop", "test-id"}]))
```

Comment:
TIL `assert_called`, awesome.

Reply:
I'm curious if calling it again would test the caching functionality (e.g. it shouldn't call the API again if calling the Repo less than an hour later). But I don't think we test anything else in that manner (I suppose we assume the caching functionality is tested elsewhere and works), so this isn't a blocking comment so much as a curiosity 😅